src/nvim/ex_getln.c

```diff
@@ -544,10 +544,14 @@ static int command_line_execute(VimState *state, int key)
   if (s->c == K_EVENT || s->c == K_COMMAND) {
     if (s->c == K_EVENT) {
       multiqueue_process_events(main_loop.events);
+
```

Comment:
So far, we have essentially _defined_ a `<cmd>` key mapping to behave the same way as if that command was executed by a timer, in the modes where this is possible. What is the reasoning behind starting to treat these two branches differently? Should only one be allowed to display certain messages, and if so, why?

Reply:
This `redrawcmdline` should be removed as well.